pages/Nvidia/_index.md

```diff
@@ -5,6 +5,10 @@ You can choose between the proprietary [Nvidia drivers](https://wiki.archlinux.o
 
 You may want to use the proprietary Nvidia drivers in some cases, for example: if you have a new Nvidia GPU model, if you want more performance, if you want to play video games, if you need a wider feature set (for example, better power consumption on recent GPUs), etc. However, keep in mind that if the proprietary Nvidia drivers do not work properly on your computer, the Nouveau driver might work fine while not having as much features or performance. For [older cards](https://wiki.archlinux.org/title/NVIDIA#Unsupported_drivers), in order to use Hyprland, you will probably need to use the Nouveau driver which actively supports them.
 
+{{< hint type=warning >}}
+All of the tips are intended to work with `nvidia`, `nvidia-open` or `nvidia-dkms` and probably won't work with the legacy or Nouveau drivers.
```

Comment:
That is good. Although, the word "probably" is unaccurate. They do not work for sure:
- They do not work on Nouveau. This driver is a completely different beast.
- They do not work on the current legacy drivers. In the future, with new legacy drivers, the will work; but for the time being, they do not. (Nvidia started barely supporting Wayland on their drivers in late 2021; and the most current legacy driver date from early 2020).

Thanks.